cvat/schema.yml

```diff
@@ -7963,10 +8062,9 @@ components:
           - $ref: '#/components/schemas/RoleEnum'
           readOnly: true
         invitation:
-          type: string
-          readOnly: true
-          nullable: true
+          $ref: '#/components/schemas/BasicInvitation'
```

Comment:
This change looks backwards-incompatible. Is there a way to make it in a backwards-compatible way?

Reply:
There is a way - to request each invitation individually on memberships page, but this leads to doing 10-15 requests, wich is not really good. 
BTW I dont think this field was used in any way, because all the previous invitations were auto accepted.

Reply:
I was thinking more along the lines of leaving this field as it is, and adding a new field with the `BasicInvitation` type.

> BTW I dont think this field was used in any way, because all the previous invitations were auto accepted.

Even if it wasn't used by the UI, it was still exposed. In particular, the SDK will attempt to parse all fields in a response, even if you don't use some of them, so this change will break it.

Reply:
Ok. I've reverted that change.
Personally, I dont like adding new field like `invitation_data` with invitation details, having a couple of invitation fields looks inconvenient. So my suggestion will be keep it as it was for now.